src/render/Cargo.toml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 The Gfx-rs Developers.
+# Copyright 2017 The Gfx-rs Developers.
```

Comment:
You sure? :)

Reply:
you got me! Need to adjust that time machine a bit, having off by 2 errors :P